### PR TITLE
cmospwd: init at 5.1

### PR DESCRIPTION
**Pull request by t4ccer**
## Description of changes

Tracked in #81418

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/security/cmospwd/default.nix [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{ lib, fetchurl, stdenv }:
```
Comment (fabaff) [inline pkgs/tools/security/cmospwd/default.nix:1]: One per line will keep the diff readable in the future.

```suggestion
{ lib
, fetchurl
, stdenv
):
```

File: pkgs/tools/security/cmospwd/default.nix [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    install -Dm0755 cmospwd -t "$out/bin"
+    runHook postInstall
+  '';
+
+  meta = {
+    description = "Decrypt password stored in cmos used to access BIOS SETUP";
+    homepage = "https://www.cgsecurity.org/wiki/CmosPwd";
+    license = lib.licenses.gpl2Plus;
+    maintainers = with lib.maintainers; [ t4ccer ];
+    platforms = lib.platforms.linux;
```
Comment (fabaff) [inline pkgs/tools/security/cmospwd/default.nix:32]:
```suggestion
  meta = with lib; {
    description = "Decrypt password stored in cmos used to access BIOS SETUP";
    homepage = "https://www.cgsecurity.org/wiki/CmosPwd";
    license = licenses.gpl2Plus;
    maintainers = with maintainers; [ t4ccer ];
    platforms = platforms.linux;
```

**Review by fabaff — COMMENTED** [review]
One concern is that `cmospwd` has not been maintained for around 15 years (the content in the tarball is dated from 2008).

**Comment by t4ccer** [comment]
I know that `cmospwd` has not been updated in a while, but it has no dependencies and requires just stdenv toolchain to build, so I think that's fine. Most importantly - it still works just fine

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 249457` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>cmospwd</li>
  </ul>
</details>

**Review by fabaff — APPROVED** [review]
- [x] package path fits guidelines
- [x] package name fits guidelines
- [x] package version fits guidelines
- [x] `meta.description` is set and fits guidelines
- [x] `meta.license` fits upstream license
- [x] `meta.maintainers` is set
- [x] package builds (see https://github.com/NixOS/nixpkgs/pull/249457#issuecomment-1685081708)
- [ ] all depending packages build
- [x] tested on x86_64

```bash
...ew/pr-249457-1]$ sudo ./results/cmospwd/bin/cmospwd 
CmosPwd - BIOS Cracker 5.1, Februar 2008, Copyright 1996-2008
GRENIER Christophe, grenier@cgsecurity.org
http://www.cgsecurity.org/

Keyboard : US
Acer/IBM                     [S  D   ][       ]
AMI BIOS                     [#  Yoo]
AMI WinBIOS (12/15/93)       [9     ]
AMI WinBIOS 2.5              [      ][][][]
[...]
```

Looks good to me.

**Pull request merged**